taqueria-vscode-extension-web-ui/src/AllEditors.tsx

```diff
@@ -0,0 +1,369 @@
+import React from 'react';
+import './AllEditors.css';
+import { MichelineEditor } from './data-editors/MichelineEditor';
+import { usePageTitle } from './hooks';
+import { MichelineEditorMessageHandler } from './interopTypes';
+
+const allTypes = [
```

Comment:
Could we add a comment here to annotate what this array of types is for, please?

Reply:
I renamed it to `typesForTestingEditor`